server/src/main.rs

```diff
@@ -0,0 +1,45 @@
+use actix_files as fs;
+use actix_web::{middleware, web, App, Error, HttpRequest, HttpResponse, HttpServer};
+use actix_web_actors::ws;
+use actix::{Actor, Addr};
+
+mod game;
+mod engine;
+
+async fn ws_index(r: HttpRequest, stream: web::Payload, data: web::Data<Addr<engine::Server>>) -> Result<HttpResponse, Error> {
+    ws::start(engine::WebSocket::new(data.get_ref().clone()), &r, stream)
+}
+
+fn env_or_default(env_name: &str, default: &str) -> String {
+    if let Ok(env_value) = std::env::var(env_name) {
+        env_value
+    } else {
+        default.to_string()
+    }
+}
+
+#[actix_rt::main]
+async fn main() -> std::io::Result<()> {
+    std::env::set_var("RUST_LOG", "debug,actix_server=info,actix_web=info");
+    let addr = env_or_default("LISTEN_ADDR", "127.0.0.1");
```

Comment:
for now it's probably fine but when the set of env vars gets larger it might be nice to have a config object with `let config = Config::from_env()` factory

Reply:
Same as above. See issue #2